metricbeat/module/system/load/load.go

```diff
@@ -0,0 +1,46 @@
+// +build darwin freebsd linux openbsd windows
```

Comment:
If load is not available on Windows then remove the `windows` build tag. You'll also want to do the same for any tests.